Core/Data.EF/Repositories/BaseRepository.cs

```diff
@@ -0,0 +1,81 @@
+﻿using AutoMapper.QueryableExtensions;
+using Lens.Core.Data.EF.Entities;
+using Lens.Core.Data.Repositories;
+using Lens.Core.Lib.Services;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Threading.Tasks;
+
+namespace Lens.Core.Data.EF.Repositories;
+
+public abstract class BaseRepository<TDbContext> : IRepository<TDbContext>
```

Comment:
Similar methods are already implemented in the DataService class. Also there are other extension methods in the files: EntityToModelExtensions.cs, EntityExtensions.cs that can be used apart of the DataService.
On the Allround project we prefer to implement a DataService for our business services and use only what is necessary. For instance, a specific service might only use get methods, so we only use the DataService.Get methods instead of implemented other unnecessary base CRUD methods.

Reply:
I guess one doesn't bite the other right? Surely something to be mindfull of.